### DISCUSSION TITLE
ComfyUI and FLUX model locations

### DISCUSSION THREAD

**Discussion by JammersUK** [comfyanonymous/ComfyUI#5955] [Q&A] [answered]
Is there a list of where FLUX model files should be? Over time I have managed to download a lot of FLUX models, through ComfyUI and externally.  But I have a lot of duplicates and no idea which goes where. The locations are: ComfyUI\models\checkpoints\FLUX1, ComfyUI\models\unet\FLUX1 and ComfyUI\models\diffusion_models\FLUX1. Will it just be easier to delete them all and let ComfyUI redownload and store them?  Thanks for any help/advice.

**Comment by LukeG89** [comment, marked as answer]
All models you have in `unet\FLUX1` folder can be moved to `diffusion_models\FLUX1`, since they are treated by ComfyUI as the same folder (`diffusion_models` was created to replace `unet`).
You can simply delete the duplicated files in unet folder if you have the same files in diffusion_models folder.

About the checkpoints, they are usually all-in-one models that can contain Diffusion, Text Encoders and VAE models inside, so you need to keep them in the checkpoint folder.

**Comment by JammersUK** [comment]
@LukeG89 Thank you for your help. I have now managed to slim down my models folder. AI uses a lot of hard drive space. 15+GB per model, builds up fast.  Again, thank you.
